test/functional/hostgroup/create_test.rb

```diff
@@ -28,7 +28,7 @@ module HammerCLIForeman
         api_expects(:hostgroups, :create) do |p|
           p['hostgroup']['content_view_id'] == 1 && p['hostgroup']['name'] == 'hg1'
         end
-        run_cmd(%w(hostgroup create --name hg1 --content-view cv1 --content-view-organization-id 1))
+        run_cmd(%w(hostgroup create --name hg1 --content-view cv1 --query-organization-id 1))
```

Comment:
Note this new command syntax for creating/updating hostgroups, as recommended by @bbuckingham